sunpy/timeseries/sources/eve.py

```diff
@@ -215,8 +222,8 @@ def peek(self, column=None, **kwargs):
 
         Parameters
         ----------
-        column : `str`, optional
-            The column to display. Defaults to ``None``, so it will display all.
+        columns : `str`, optional
```

Comment:
So this is a string but the plot method wants a list of strings, we should unify this somehow.